brem/AsyncCopyOverSSH.py

```diff
@@ -0,0 +1,186 @@
+from PySide2 import QtCore
+from eqt.threading import Worker
+import paramiko
+from brem import BasicRemoteExecutionManager
+import os, ntpath, posixpath
+import socket
+
+class RemoteAsyncCopyOverSSHSignals(QtCore.QObject):
+    status = QtCore.Signal(tuple)
+    job_id = QtCore.Signal(int)
+class AsyncCopyOverSSH(object):
+    '''Class to handle async copy of files over SSH'''
```

Comment:
expand on the docstring. Add `remote_os` in the parameter for the creator as [_often-changed_](https://github.com/TomographicImaging/CIL/discussions/1064#discussioncomment-1657282).